### ISSUE TITLE
create `standard-readme` layout

### ISSUE THREAD

**Issue by tunnckoCore** [verbose/verb-generate-readme#10]
I'm going to major standardizing and my flow would soon be totally automated, crazy and awesome. I'm using the "standards" - `standard`, semantic releasing with `standard-version` (which is almost the same as `semantic-release`), and wanna have standard readmes too.

offtopic: What I imagine:
1. generate-repo - creates repo on github, labels etc, init and possibly first commit
2. generate project - creates any templates
3. writing the modules
4. verb
5. git add --all -> git cz (commit using commitizen), (standard-version), git push, npm publish

**Comment by jonschlinkert** [comment]
Yeah, I thought it was funny that you grabbed the `standards` org the day I dropped it. (How did you know I dropped btw?)

> standard-readme

What does this mean? I'm not sure what this issue is asking for specifically

fwiw, this is all something that should be customizable by the user. We have always planned on removing most of the default templates. So I probably wouldn't want to add more. I'd rather focus on fixing whatever is preventing you from customizing the way you want

**Comment by tunnckoCore** [comment]
> Yeah, I thought it was funny that you grabbed the standards org the day I dropped it. (How did you know I dropped btw?)

Ahahah, actually, didn't know, but in most of the times we think in same way and do similar things at same time - i noticed this a couple of times, starting before 2 years or so.. It's incredible. It was the same case with @es128's `glob-parent` and http://ghub.io/glob2fp haha - diff 1-2 days. Anyway.

> fwiw, this is all something that should be customizable by the user.

Yea, I know, but i'm thinking it would be good thing to have builting at least 3 layouts: one "standard" readme, empty and yours - nothing more. Where your current readmes are near to the spec, so it also would be great if you fix them a bit to be compliant.

**Comment by jonschlinkert** [comment]
> so it also would be great if you fix them a bit to be compliant.

How about if we fix the spec instead

**Comment by tunnckoCore** [comment]
I'm not forcing you to do it :laughing:. I'm just asking to have builtin layout which users can use without doing so much and this will help `standard-readme` to grow. :)

**Comment by jonschlinkert** [comment]
I looked at standard-readme, there is nothing there...? What am I missing?

**Comment by tunnckoCore** [comment]
https://github.com/RichardLitt/standard-readme? `spec.md`

**Comment by jonschlinkert** [comment]
was he one of the people in that discussion about standardizing readmes?

**Comment by tunnckoCore** [comment]
Yea.

**Comment by jonschlinkert** [comment]
I just looked. I have zero interest in supporting that. I tried commenting in the one discussion and the response(s) to my comment were childish, and it seemed like they were actively discouraging others from commenting. Someone even commented that they didn't like my style of readmes. That's fine, but given that I have more of them, more experience writing them, and more tools for creating them than any other GitHub user to my knowledge, I would have at least hoped to be a part of the discussion. 

I even commented that my decisions are based on data, and someone had a problem with that. It was lame. I have 300 million downloads a month to guide my decisions, not to mention the clicks that we track. It's absurd for someone to not want insight from that.

**Comment by RichardLitt** [comment]
@jonschlinkert hey, really sad to hear that was your experience. Can you point to that conversation? I don't recall seeing your comment, and I am 100% in favor of data-driven workflows.

**Comment by jonschlinkert** [comment]
@RichardLitt thanks for the comment. I now can't recall where that was, but I'll add the link here if I remember.
